src/kernels/MIOpenRNNHiddenStateUpdate.cl

```diff
@@ -2,7 +2,7 @@
  *
  * MIT License
  *
- * Copyright (c) 2019 Advanced Micro Devices, Inc.
+ * Copyright (c) 2020 Advanced Micro Devices, Inc.
```

Comment:
The license header includes all years one has modified the file.  So, you should specify 2019-20 as 2019 was the year when it was first authored.

Reply:
@daniellowell Hey Daniel, what is the license format we are using now?

Reply:
@daniellowell Please provide your input. In past, the licenses I have seen includes all the years since the file was authored.

Reply:
Year the file was authored.